src/containers/Home/OurPartnersSection/OurPartnersSection.js

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+import routes from '../../../constants/routes';
+
+import ServiceTile from './ServiceTile';
 import styles from './OurPartnersSection.module.css';
 
+import safeImage from './SAFe.png'
+import scrumOrgImage from './Scrum-org.png'
+import scrumAllianceImage from './Scrum-alliance.png'
+
+
+
 const OurPartnersSection = () => {
-  return (
+    return (
     <section className={styles.our_partners_section}>
-      <h3>Our Partners</h3>
-      <div>
-        <Link to={"#"}>SAFe</Link>
-        <Link to={"#"}>Scrum Alliance</Link>
-        <Link to={"#"}>Scrum.org</Link>
-      </div>
+	  <h3>Our Partners</h3>
+            <div className={styles.tiles_row}>
+
+            <ServiceTile
```

Comment:
```suggestion
            <ServiceTile
                src={safeImage}
                url={'https://www.scaledagileframework.com/'}
            >
```
Here we should pass the url in as a prop to the `<ServiceTile>` component.